FND/Utilities/Analytics.rdf

```diff
@@ -738,6 +761,12 @@ For certain indices, one of the most common weighting factor is by market capita
 		<skos:definition>indicates the number of elements in a given population</skos:definition>
 	</owl:DatatypeProperty>
 	
+	<owl:DatatypeProperty rdf:about="&fibo-fnd-utl-alx;hasReferenceDateTime">
```

Comment:
not sure about "Reference" here which could get confused with "Base".

Reply:
See above on the renaming discussion.